GUI/QtQuick/qml/TransactionsPage.qml

```diff
@@ -168,6 +168,23 @@ Item {
             onCopyRequested: bsApp.copyAddressToClipboard(id)
 
             onCellClicked: (row, column, data) => {
+                show_transaction_details(row, column, data)
+            }
+
+            onCellRightClicked: (row, column, data) => {
+                var nbConf = model.data(model.index(row, 5), TxListModel.NbConfRole)
+                if (nbConf === 0)
+                {
+                    show_rbf_cfpf_menu(row, column, data)
+                }
+                else
+                {
+                    show_transaction_details(row, column, data)
+                }
+            }
+
+            function show_transaction_details(row, column, data)
+            {
```

Comment:
This looks like a copy-paste from OverviewPanel. Can we have only one instance of code?

Reply:
Yes I will create CistomTransactionsTable.qml - shared to both tables. It seems like different only width, height and model